internal/premis/premis.go

```diff
@@ -128,30 +129,23 @@ func eventFromEventSummaryAndAgent(eventSummary EventSummary, agent Agent) Event
 	}
 }
 
-func NewEventSummary(eventType string, failures []string) (EventSummary, error) {
-	detail, outcome := assembleEventDetailAndOutcome(failures)
+func NewEventSummary(eventType string, detail string, outcomeDetail string, failures []string) (EventSummary, error) {
```

Comment:
Do we still need to pass the slice of failures, could we use a boolean or pass the outcome directly.

Reply:
Good call... that'd make it more straightforward.

Reply:
(Fixed.)